cmd/scollector/collectors/processes_windows.go

```diff
@@ -10,31 +10,28 @@ import (
 	"bosun.org/opentsdb"
 )
 
-var regexes = []*regexp.Regexp{}
+var regexesProcesses = []*regexp.Regexp{}
 
 func AddProcessConfig(line string) error {
 	reg, err := regexp.Compile(line)
```

Comment:
I agree with your agreement as long as the program exits when it encounters a regex compile error and doesn't just log it.